.github/workflows/chromatic.yml

```diff
@@ -1,26 +1,22 @@
 # .github/workflows/chromatic.yml
 
-# Workflow name
 name: 'Chromatic'
 
-# Event for the workflow
-on: push
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
So maybe only on push to master?
Since it will run twice for both opened and pushed when a PR is created & twice for both synchronize and push when a commit is pushed

Reply:
`push` is not running for forks that's reason this workflow currently doesn't run on the PRs of external contributors

About the labels - good trick, but allowing it on `pull_request` seems secure and same time easier, no? You think it might be a security breach?

Reply:
I don't think it's a security breach, it would just run twice like many of our actions that we should anyway fix